src/js/components/InfiniteScroll/InfiniteScroll.js

```diff
@@ -177,7 +177,7 @@ const InfiniteScroll = ({
       }
     }, 100);
     return () => clearTimeout(timer);
-  }, [renderPageBounds, show, step]);
+  }, [renderPageBounds, show, step, scrollShow]);
```

Comment:
I'm concerned that will have unintended side effects, as this effect calls `setScrollShow`, which will cause `scrollShow` to change. Since this is already a warning today. I think we should decouple this from the webpack upgrade pull request, so that it can get more attention and not hold the upgrade up.

Reply:
Ok makes sense I removed this change in the latest commit

Reply:
I've created a separate issue for this warning as it seems to come back on other PR's as well.